CMakeLists.txt

```diff
@@ -57,11 +57,41 @@ set(public_include_header_list
   "${library_root}/translate.hh"
   "${library_root}/types.h"
   "${library_root}/xml.hh"
+  "${library_root}/architecture.hh"
+  "${library_root}/capability.hh"
+  "${library_root}/stringmanage.hh"
+  "${library_root}/fspec.hh"
+  "${library_root}/comment.hh"
+  "${library_root}/pcodeinject.hh"
+  "${library_root}/rangeutil.hh"
+  "${library_root}/printlanguage.hh"
+  "${library_root}/prettyprint.hh"
+  "${library_root}/cast.hh"
+  "${library_root}/prefersplit.hh"
+  "${library_root}/op.hh"
+  "${library_root}/options.hh"
+  "${library_root}/transform.hh"
+  "${library_root}/cpool.hh"
+  "${library_root}/typeop.hh"
+  "${library_root}/action.hh"
+  "${library_root}/emulateutil.hh"
+  "${library_root}/jumptable.hh"
+  "${library_root}/block.hh"
+  "${library_root}/varmap.hh"
+  "${library_root}/varnode.hh"
+  "${library_root}/cover.hh"
+  "${library_root}/rangemap.hh"
+  "${library_root}/type.hh"
+  "${library_root}/database.hh"
+  "${library_root}/variable.hh"
+  "${library_root}/userop.hh"
```

Comment:
I also found that we were missing all of these headers recently 😬

Reply:
I have a PR here #107

Reply:
That's now been merged in. I'll resolve conflicts on this branch.